docs/pages/validator-management/vc-configuration.md

```diff
@@ -88,11 +88,29 @@ Configure your validator client's fee recipient address by using the `--suggeste
 
 You may choose to use the `--strictFeeRecipientCheck` flag to enable a strict check of the fee recipient address with the one returned by the beacon node for added reassurance.
 
+### Configure your builder selection and/or builder boost factor
+
+If you are running a beacon node with connected builder relays, you may use these validator configurations to signal which block (builder vs. local execution) the beacon node should produce.
+
+With produceBlockV3 (enabled automatically after the Deneb hard fork), the `--builder.boostFactor` is a percentage multiplier the block producing beacon node must apply to boost (>100) or dampen (<100) builder block value for selection against execution block. The multiplier is ignored if `--builder.selection` is set to anything other than `maxprofit`. Even though this is set on the validator client, the calculation is applied on the beacon node itself. For more information, see the [produceBlockV3 Beacon API](https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/produceBlockV3).
+
+With Lodestar's `--builder.selection` options, you can select:
+- `maxprofit`: Default setting for Lodestar set at `--builder.boostFactor=100`. This default setting will always choose the more profitable block. Using this option, you may customize your `--builder.boostFactor` to your preference. Examples of its usage are below.
+- `executiononly`: Validator will not select the builder block even if builder relays are configured on the beacon. This option will always select the local execution block.
```

Comment:
```suggestion
- `executiononly`: Beacon will not produce/select the builder block even if builder relays are configured on the beacon. This option will always select the local execution block and will error if it couldn't produce one.
```

Reply:
This is what Lodestar beacon node does but we should ideally describe how the validator client behaves here as this configuration page related to the vc configuration.

Reply:
I have updated the description based on the suggestion https://github.com/ChainSafe/lodestar/pull/6295/commits/9e329d45bab7db8f9ea707de1fc8a146642adae1 (with slight modification)